flexbe_states/CMakeLists.txt

```diff
@@ -25,6 +25,14 @@ catkin_package(
 #install(PROGRAMS bin/hello
 #        DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
 
+install(CODE "
+  file(MAKE_DIRECTORY \"$ENV{DESTDIR}${CMAKE_INSTALL_PREFIX}/${CATKIN_PACKAGE_SHARE_DESTINATION}/src\")
```

Comment:
Respecting the `DESTDIR` environment variable is required for Debian-packaging (https://www.gnu.org/prep/standards/html_node/DESTDIR.html).